Software/Core/src/main/java/org/foi/emp/core/Entities/ElementModelaPracenja.java

```diff
@@ -0,0 +1,20 @@
+package org.foi.emp.core.Entities;
+
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
```

Comment:
Is this going to be inserted in local database?
Something is missing.
`@Entity(tableName = "tableName")`